addon/content/reducer.js

```diff
@@ -396,7 +396,8 @@ function messages(state = initialMessages, action) {
     }
     case "TAG_CLICK": {
       const msg = Conversations.currentConversation.getMessage(action.msgUri);
-      msg.msgPluginTagClick(topMail3Pane(window), action.event, action.detail);
+      const { event, details } = action;
+      msg.msgPluginTagClick(topMail3Pane(window), event, details);
```

Comment:
I'm not quite sure we need this breakdown, if it was more fields then maybe. However, this feels a little unnecessary.